data/json/recipes/food/seed.json

```diff
@@ -1,5 +1,161 @@
+        "components": [ [ [ "potato_raw", 1 ] ] ]
+    },
+    {
+        "result": "seed_oats",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "oats", 1 ] ] ]
+    },
+    {
+        "result": "marloss_seed",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "marloss_berry", 1 ] ] ]
+    },
+    {
+        "result": "seed_corn",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "corn", 1 ] ] ]
+    },
+    {
+        "result": "seed_carrot",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "carrot", 1 ] ] ]
+    },
+    {
+        "result": "seed_onion",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "onion", 1 ] ] ]
+    },
+    {
+        "result": "seed_zucchini",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "zucchini", 1 ] ] ]
+    },
+    {
+        "result": "seed_sugar_beet",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "sugar_beet", 1 ] ] ]
+    },
+    {
+        "result": "seed_barley",
+        "type": "recipe",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_OTHER",
+        "skill_used": "survival",
+        "difficulty": 2,
+        "time": 5000,
+        "autolearn": true,
+        "qualities": [ { "id": "CUT", "level": 1 } ],
+        "components": [ [ [ "barley", 1 ] ] ]
+    },
```

Comment:
I will ask my Horticulturist wife about this one, but I think they cook the barley before they put it in stores, or before you can eat it, and that renders the barley unplantable.